src/AcceptanceTests/Routing/When_subscribing_to_a_derived_polymorphic_event_with_forwarding_topology.cs

```diff
@@ -4,18 +4,18 @@
     using System.Threading.Tasks;
     using AcceptanceTesting;
     using AcceptanceTesting.Customization;
-    using AzureServiceBus;
     using NServiceBus.AcceptanceTests;
     using NServiceBus.AcceptanceTests.EndpointTemplates;
     using Features;
-    using Settings;
     using NUnit.Framework;
 
     public class When_subscribing_to_base_and_derived_polymorphic_events_with_forwarding_topology : NServiceBusAcceptanceTest
     {
         [Test]
         public async Task Should_handle_each_event_once_only()
         {
+            RequiresAdditionally.ForwardingToplogy();
```

Comment:
Now that I look at it, `RequiresAdditionally` is a bit misleading. It **requires** it and period. Only when it's the right topology, then the test can run. Rather see `Requires.ForwardingToplogy()`

Reply:
Or `TestRequires.ForwardingToplogy()`

Reply:
Requires is already taken. I can rename to TestRequires if you prefer